Formula/httpd.rb

```diff
@@ -37,6 +211,9 @@ def install
                           "--enable-mpms-shared=all",
                           "--enable-mods-shared=all",
                           "--enable-pie",
+                          "--enable-suexec",
+                          "--with-suexec-bin=#{opt_bin}/suexec",
+                          "--with-suexec-caller=_www",
```

Comment:
What are these needed for?

Reply:
These are for the suEXEC module that allows CGI and SSI scripts to be run as different users than the server process (something often seen in shared hosting environments). Because of the risks with using the module, it's not build without explicit enable, as was in before the migration.

Reply:
I think I'd rather pass on it as we're not a shared hosting environment here unless you're happy to add it without the caveats.

Reply:
Let's pass on this for now and split root/suexec stuff into another PR so this one can get merged quicker.